core/src/commonMain/kotlin/Channels.kt

```diff
@@ -0,0 +1,8 @@
+package com.juul.kable
+
+import kotlinx.coroutines.channels.SendChannel
+
+// Workaround until https://github.com/Kotlin/kotlinx.coroutines/issues/974 is closed.
```

Comment:
Should we mark this with TODO to make sure we know to check up on coroutines' issue?

Reply:
In this particular case, I don't feel `TODO` is necessary, since the workaround will exhibit desired behavior (i.e. there isn't a bug remaining after this change — there will just be a way to do it in the future that requires less code, after issue 974 is closed) and it would be fine if we never changed it.

**tl;dr** I don't think there's a need for an explicit marker of work to be done.